### PR TITLE
chore: change commit message

### PR DESCRIPTION
**Pull request by JoeWang1127**
In this PR:
- Change commit message to include source link.

### CODE REVIEW

File: library_generation/test/resources/integration/google-cloud-java/pr-description-golden.txt [hunk 1]
```diff
@@ -1,23 +1,19 @@
 This pull request is generated with proto changes between googleapis commit a17d4caf184b050d50cacf2b0d579ce72c31ce74 (exclusive) and 4ce0ff67a3d4509be641cbe47a35844ddc1268fc (inclusive).
```
Comment (blakeli0) [inline library_generation/test/resources/integration/google-cloud-java/pr-description-golden.txt:1]: Can we make these two commit a link too?

File: library_generation/utils/commit_message_formatter.py [hunk 2]
```diff
@@ -47,19 +47,24 @@ def format_commit_message(commits: Dict[Commit, str], is_monorepo: bool) -> List
...
 
+    :param commit: a GitHub commit.
     :param messages: a (multi-line) commit message, one line per item.
     :return: wrapped messages.
     """
+    short_sha = commit.hexsha[:7]
     result = ["BEGIN_NESTED_COMMIT"]
     result.extend(messages)
+    result.append(
+        f"Source Link: [googleapis/googleapis@{short_sha}](https://github.com/googleapis/googleapis/commit/{commit.hexsha})"
```
Comment (diegomarquezp) [inline library_generation/utils/commit_message_formatter.py:66]: Optional nit:

FWIW Github will automatically format the link if you just write the link in plain text. That way we make the logic simple at the cost of relying on the service (is it stable?).

E.G. pasting `https://github.com/googleapis/googleapis/commit/19af522bf3ab49b7d4ce284a568b2e8bb28aeae3` results in https://github.com/googleapis/googleapis/commit/19af522bf3ab49b7d4ce284a568b2e8bb28aeae3

**Review by diegomarquezp — APPROVED** [review]
Thank you

File: library_generation/utils/commit_message_formatter.py (hunk 2 above)
Comment (JoeWang1127) [reply library_generation/utils/commit_message_formatter.py:66]: Thanks for the suggestion, though I don't want to depends on github.

File: library_generation/test/resources/integration/google-cloud-java/pr-description-golden.txt (hunk 1 above)
Comment (JoeWang1127) [reply library_generation/test/resources/integration/google-cloud-java/pr-description-golden.txt]: Done.

**Review by blakeli0 — APPROVED** [review]

**Pull request merged**